bucket/corporateclash.json

```
@@ -0,0 +1,17 @@
+{
+  "version": "1.2.0",
+  "description": "Toontown: Corporate Clash is a completely free to play massively multiplayer online game designed to be the new experience of a game many of us loved. This is a game designed for players of all ages. ANYONE can have fun joining in the battle against the evil robot cogs. So, grab a seltzer and a few pies and dive right into the never ending battle to save Toontown from the cogs! What are you waiting for?",
+  "homepage": "https://corporateclash.net",
+  "hash": "bf546c37691ca14c27c6dbb186efd5100553b42bcb72f0f8829064fa74aa67f6",
```

The checksum listed here is incorrect. Right now, I'm getting this SHA-256 hash for the download URL:

```suggestion
  "hash": "0b764e17e9385ab5dc5be958f63377480b3df001c5762471172a873b7fcef057",
```

If the hash changes frequently without notice, I recommend setting the version identifier to `nightly` and removing the `hash` entry from the manifest.